### PR TITLE
Adding an 'exact_text' option for getting exactly the text you want in ambiguous results

### PR DESCRIPTION
**Pull request by garymh**
Hi - not sure if this is an option you want, but I needed it for a project so thought I'd pass it along 🙂

### CODE REVIEW

**Comment by Hirurg103** [comment]
Hi @garymh , thank you for your PR. I will review it later

**Comment by Hirurg103** [comment]
Will try to review this week

File: lib/capybara_select2/helpers.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@ def select2(*args)
 
         find_options = { text: value }
         find_options[:match] = options[:match] if options[:match]
+        find_options[:exact_text] = options[:exact_text] if options[:exact_text]
```
Comment (Hirurg103) [inline lib/capybara_select2/helpers.rb]: how about

```ruby
find_options = options.slice(:match, :exact_text).merge({ text: value })
```
?

Comment (garymh) [reply lib/capybara_select2/helpers.rb]: Absolutely! That makes it look much cleaner.

**Comment by Hirurg103** [comment]
Actually could you please revert the last commit (maybe better to hard reset the last commit and force-push) it as seems like `Hash#slice` is a method which is being added by `ActiveSupport` library. There is no such method in a [plain-ruby Hash](https://docs.ruby-lang.org/en/2.0.0/Hash.html)

git commands to reset the commit and force push (Just in case)
```
git reset --hard HEAD~1
git push -f origin master
```

That's why the build is red.

**Comment by Hirurg103** [comment]
Thank you @garymh for your work on this improvement!

**Pull request merged**